cosvm/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cosvm",
+  "assets": [
+    {
+      "description": "",
+      "denom_units": [
+        {
+          "denom": "ucvm",
+          "exponent": 0
+        },
+        {
+          "denom": "cvm",
+          "exponent": 6
+        }
+      ],
+      "base": "ucvm",
+      "name": "CosVM",
+      "display": "cvm",
+      "symbol": "CVM",
+      "coingecko_id": "cosvm",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosvm/images/cvm.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosvm/images/cvm.svg"
```

Comment:
Please do not upload false SVG (a png exported as a svg)--as it is not scalable.